src/pages/debrief/cat-a-mod2/__tests__/debrief.cat-a-mod2.page.spec.ts

```diff
@@ -0,0 +1,292 @@
+import { ComponentFixture, async, TestBed } from '@angular/core/testing';
+import { IonicModule, NavController, NavParams, Config, Platform } from 'ionic-angular';
+import { NavControllerMock, NavParamsMock, ConfigMock, PlatformMock } from 'ionic-mocks';
+
+import { AppModule } from '../../../../app/app.module';
+import { DebriefCatAMod2Page } from '../debrief.cat-a-mod2.page';
+import { AuthenticationProvider } from '../../../../providers/authentication/authentication';
+import { AuthenticationProviderMock } from '../../../../providers/authentication/__mocks__/authentication.mock';
+import { DateTimeProvider } from '../../../../providers/date-time/date-time';
+import { DateTimeProviderMock } from '../../../../providers/date-time/__mocks__/date-time.mock';
+import { By } from '@angular/platform-browser';
+import { ComponentsModule } from '../../../../components/common/common-components.module';
+import { StoreModel } from '../../../../shared/models/store.model';
+import { StoreModule, Store } from '@ngrx/store';
+import {
+  AddDangerousFault,
+} from '../../../../modules/tests/test-data/common/dangerous-faults/dangerous-faults.actions';
+import { AddSeriousFault } from '../../../../modules/tests/test-data/common/serious-faults/serious-faults.actions';
+import { AddDrivingFault } from '../../../../modules/tests/test-data/common/driving-faults/driving-faults.actions';
+import {
+  EyesightTestFailed,
+  EyesightTestPassed,
+} from '../../../../modules/tests/test-data/common/eyesight-test/eyesight-test.actions';
+import { Competencies } from '../../../../modules/tests/test-data/test-data.constants';
+import { DebriefComponentsModule } from '../../components/debrief-components.module';
+import { ScreenOrientation } from '@ionic-native/screen-orientation';
+import { Insomnia } from '@ionic-native/insomnia';
+import { InsomniaMock } from '../../../../shared/mocks/insomnia.mock';
+import { ScreenOrientationMock } from '../../../../shared/mocks/screen-orientation.mock';
+import { TranslateModule, TranslateService } from 'ng2-translate';
+import { fullCompetencyLabels } from '../../../../shared/constants/competencies/catb-competencies';
```

Comment:
I would think this will need change to CatAMod2 competencies. Todo for now should be fine.